app/assets/images/link.svg

```diff
@@ -1,3 +1,3 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="482.136" height="482.135" viewBox="0 0 482.136 482.135">
```

Comment:
We've had this old asset that hasn't been used anyway so I just replaced it with new one.